test/test.py

```diff
@@ -104,9 +127,28 @@ def test_external_scripts(self):
 
         self.assertEqual(output, 'the version is 1.2.0')
 
+    def test_subproject(self):
+        clang_build_try_except(['-d', 'test/subproject', '-V', '-p'])
+
+        try:
+            output = subprocess.check_output(['./build/mainproject/default/bin/runLib'], stderr=subprocess.STDOUT).decode('utf-8').strip()
+        except subprocess.CalledProcessError:
+            self.fail('Could not run compiled program')
+
+        self.assertEqual(output, 'Hello! mylib::triple(3) returned 9')
+
+    # def test_openmp(self):
```

Comment:
What is this? If the openMP test does not work, the openMP stuff should be removed from the test cases.

Reply:
Analogous to the multi-target test, which currently does not work, I left it in because we should get it running at some point.